packages/datadog-plugin-aws-sdk/test/stepfunctions.spec.js

```diff
@@ -0,0 +1,148 @@
+/* eslint-disable max-len */
+'use strict'
+
+const Stepfunctions = require('../src/services/stepfunctions')
+const semver = require('semver')
+const agent = require('../../dd-trace/test/plugins/agent')
+const { setup } = require('./spec_helpers')
+
+const helloWorldSMD = {
+  Comment: 'A Hello World example of the Amazon States Language using a Pass state',
+  StartAt: 'HelloWorld',
+  States: {
+    HelloWorld: {
+      Type: 'Pass',
+      Result: 'Hello World!',
+      End: true
+    }
+  }
+}
+
+describe('Sfn', () => {
+  let tracer
+
+  describe('Injection behaviour', () => {
+    before(() => {
+      tracer = require('../../dd-trace')
+    })
+
+    it('generates tags for a start_execution', () => {
+      const sfn = new Stepfunctions(tracer)
+      const params = {
+        stateMachineArn: 'arn:aws:states:us-east-1:425362996713:stateMachine:agocs-test-noop-state-machine-2'
+      }
+      expect(sfn.generateTags(params, 'startExecution', {})).to.deep.equal({
```

Comment:
This should be validated by checking what is sent to the agent, not by calling the internal function directly.

Reply:
Same for the test below.